python/rapidsmpf/rapidsmpf/integrations/dask/core.py

```diff
@@ -344,6 +364,10 @@ def bootstrap_dask_cluster(
         memory pool raises OOM errors.
     enable_statistics
         Whether to track shuffler statistics.
+    options
+        Configuration options. This argument takes precedence: if not None, the
+        other arguments (beside `client`) are ignored!
+        TODO: remove the other arguments beside `client`.
```

Comment:
Seem like this is just to provide temporary backward compatibility, correct?

Reply:
If we can get the cudf component ready while this PR is still in review, it probably makes sense to just break the  `bootstrap_dask_cluster` API.